lib/graticule/geocoder/geonames.rb

```diff
@@ -3,12 +3,13 @@ module Graticule #:nodoc:
   module Geocoder #:nodoc:
     class Geonames < Base
 
-      def initialize
+      def initialize(username = nil)
```

Comment:
Hmm, this was 2+ months ago, but as I recall it needed to be optional to keep the GeoNames service consistent with others.

We ended up making the change in a forked version, which is what we still use. I think that's because nobody at CI was maintaining Graticule. You're welcome to pull in my change or make it another way, but anyway, if you're getting your Graticule version up-to-date, I'd be glad for us to switch back to it.